### DISCUSSION TITLE
Mouse events on Row

### DISCUSSION THREAD

**Discussion by andreparodi** [hecrj/iced#1359] [General]
Hi,

I am trying to replicate some behavior of a table where I click on a row and it gets highlighted and a panel shows the details. 

Ideally I would like to trigger a Message on_hover or on_press for a Row like a Button but I don't see any such option. I see an on_event but not sure I that is what I should try to use. 

Any pointer much appreciated
Andre

**Comment by 13r0ck** [comment]
Just put the row inside a button. Then set the default, clicked, and hover style for the button to be what you want the row to look like

A message on hover is not possible currently in iced, there are technical limitations that haven't yet been resolved.
